{"name": "登录反向用例测试", "status": "passed", "attachments": [{"name": "log", "source": "0fc9f508-1ca5-492b-ba6d-4ec1aa20ff64-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'name': '登录-反向用例-用户名错误', 'username': '财友', 'password': '654321', 'error_msg': '登录验证失败'}"}], "start": 1615361150427, "stop": 1615361153234, "uuid": "66f66051-7af6-4f2e-bc1f-d5a48833c0e6", "historyId": "28fbee3f7849e5c692210a9f12e7fe96", "testCaseId": "130e6dce60bc4ef9d882c5f0465dc1d6", "fullName": "cases.test_login.TestLogin#test_login_failed", "labels": [{"name": "epic", "value": "钓鱼人APP自动化测试"}, {"name": "feature", "value": "钓鱼人APP自动化测试-登录模块"}, {"name": "story", "value": "钓鱼人APP自动化测试-登录模块-反向用例测试"}, {"name": "tag", "value": "@pytest.mark.usefixtures('prepare_to_login')"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-LSM6I36P"}, {"name": "thread", "value": "16828-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_login"}]}